jdaviz/configs/default/plugins/export_plot/export_plot.py

```diff
@@ -237,6 +243,11 @@ def save_movie(self, i_start=None, i_end=None, fps=None, filename=None, filetype
         if not HAS_OPENCV:
             raise ImportError("Please install opencv-python to save cube as movie.")
 
+        if not self.movie_enabled:

```

Comment:
Should this check happen before HAS_OPENCV check?

Reply:
I can move it if you want, but the use-case for this switch is for standalone instances running on external servers (where the user doesn't have python-access... otherwise they can just re-enable it) in which case the button is hidden.  Without the button or python-access, this shouldn't be possible to reach.  I just added this extra check to prevent some malicious mimicking of the javascript call to create a large file on the server.

Reply:
I was thinking more like poor @havok2063 will never be able to test this without installing opencv-python first.

Reply:
The UI section should still hide itself, but if anyone wants to test the javascript hacking... then that's true.  I'll flip the order just in case (doesn't do any harm).